### DISCUSSION TITLE
关于剧集偏移功能

### DISCUSSION THREAD

**Discussion by Illidan1009** [EstrellaXD/Auto_Bangumi#615] [Q&A]
麻烦请问一下，剧集偏移功能应该怎么设置呢，修改设置能对已下载剧集生效吗？
比如rss里是直接从第25级开始的，我想设置成S02E01……，但不管我设置偏移-24还是24，重命名之后还是S02E25

**Comment by EstrellaXD** [comment]
目前这个功能还没做好，预计在下个大版本加上
